tools/picard.js

```diff
@@ -299,6 +299,8 @@ function build_ui () {
         default_width: RESOLUTIONS[2][0],
         default_height: RESOLUTIONS[2][1],
     });
+    // Add the .WindowSimple class so we get styling on the scrollbars
+    widgets.window.get_style_context().add_class('WindowSimple');
```

Comment:
Feasible yes, but do we want that? I thought those scrollbars were pretty specific to knowledge-lib. Not sure we want them on say, photo app?